### ISSUE TITLE
Angel lda* "apache/commons/math/special/Gamma" error?

### ISSUE THREAD

**Issue by WisonHuang** [Tencent/angel#237]
![171027161530](https://user-images.githubusercontent.com/6241452/32094637-11141ff2-bb33-11e7-9a80-644b1fb26195.png)

I am trying to run the lda example in YARN mode. I can succeed in running the example in LOCAL mode.But  when I change to YARN mode . I encounter the error "apache/commons/math/special/Gamma".
Here is my running script:
![image](https://user-images.githubusercontent.com/6241452/32094589-df1bc40a-bb32-11e7-9b3b-68ee0cb38363.png)

**Comment by WisonHuang** [comment]
I follow the job log and got the same error "diagnostic: apache/commons/math/special/Gamma". I solved this problem by the following , I hope someone with similiar problem can get some inspirations.
My hadoop environment jdk version is 1.7, and I put jdk 1.8 to cluster computer nodes as [](https://github.com/Tencent/angel/wiki/Angel%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98) says , to make the environment is ok.
Then I ran the lda demo, I succeeded in running it in LOCAL mode. But failed in YARN mode, and I get the error "diagnostic: apache/commons/math/special/Gamma". That confused me ,i followed the code and it seems that something is wrong with the "Gamma" class. But I can't find anything strang with the "Gamma" class. Then I saw that in the job  tracking logs, none of my workers has run ,all of them just failed in the "New" phrase. Suddenly, I came to a thought that maybe my hadoop enviroment has some diff in the "Gamma" Class, maybe hadoop env doesn't has the "Gamma" Class. So I put the "commons-math-2.2.jar" that "Gamma" class belongs to , to the hadoop env by using "angel-site.xml".
Finally ,I succeed in running the lda demo in YARN mode.
